tests/unit/services/cookies-test.js

```diff
@@ -306,6 +306,15 @@ describe('CookiesService', function() {
 
         this.subject().write(COOKIE_NAME, 'test', { path: '/sample-path', maxAge: 1000 });
       });
+
+      it('throws when a larger than allowed cookie is set', function() {
+        expect(() => {
+          let value = "";
+          for(let i = 0; i < 400; i++) { value += Math.random().toString(36).substring(2) }
```

Comment:
I guess we could just use a static value here that we know has more than 4096 bytes?